questionpy_server/worker/runtime/messages.py

```diff
@@ -195,7 +196,10 @@ class Response(MessageToServer):
 
 
 class WorkerError(MessageToServer):
-    """Error message."""
+    """Error message.
+
+    TODO: add a CUSTOM-ErrorType which the package can raise?
```

Comment:
Sehe ich gerade keinen konkreten Use-Case für. Für unerwartete technische Fehler ist `ErrorType.UNKNOWN` da, alle nicht-technischen Fehler sollten wir vorgeben, um sie dann auch gesondert zu behandeln und anzuzeigen.

Möglicherweise wäre das hilfreich für den Fall, dass die Question-Erstellung fehlschlägt, aber das nicht in den Options liegt (z.B. weil ein externer Dienst nicht erreichbar ist). Dann wäre `FORM_OPTIONS_INVALID` unpassend, aber dennoch eine lokalisierte, Domain-spezifische Fehlermeldung wünschenswert. Das ist mir aber gerade noch zu ungewiss, um über Implementierung nachzudenken.